src/Lachain.Core/Consensus/BlockProducer.cs

```diff
@@ -18,7 +18,8 @@
 using Lachain.Proto;
 using Lachain.Storage.State;
 using Lachain.Utility.Utils;
-
+using Lachain.Core.Config;
```

Comment:
try to use them in alphabetical order please

Reply:
Updated in the latest commit.